JULIERAJ/KIDS-FIRST-Front
# Feat/134/use delete message modal in kidform

## One Line Description
Integrate reusable confirmation modal in KidForm

## Test Steps
1. Open the application.
2. Navigate to the sidebar and click on "Kid's Info".
3. On the Kid's Info page, click "Add Your Kid".
4. In the kid form, click on the Image field. A dropdown will appear.
5. In the dropdown, click on "Remove Photo". A confirmation message will open.

## UI/UX Outcome
- ![confirmMessage](https://github.com/user-attachments/assets/8d89ba00-4c9f-4eb2-8b42-c598e88d8fff)
[Insert relevant screenshots or videos here]

## Checklist
